buy_together_app/tests/test_sign_up.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from django.template.loader import get_template
+from supplier.models import Supplier
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db()
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template('buy_together_app/sign_up.html')
```

Comment:
What are we testing here?

Reply:
Aren't we testing the function that actually makes use of the template?

Reply:
Both works but i will be adding a test to test the get page request for /sign 
up

Reply:
```suggestion
    def test_signup_template_exists(self):
        get_template('buy_together_app/sign_up.html')
```